frontend/javascripts/oxalis/view/action-bar/tracing_actions_view.tsx

```diff
@@ -614,14 +614,12 @@ class TracingActionsView extends React.PureComponent<Props, State> {
       );
     }
 
-    if (restrictions.allowSave) {
```

Comment:
I was curious why this condition was introduced (since the version view seems to work just fine for read-only annotations) and checked using git blame. Turns out, I introduced this when sandbox annotations were implemented. For sandbox annotations, the version view doesn't work, it cannot be closed again, for example :see_no_evil: You can check using a sandbox URL, like https://allowversionhistoryinnotownedannotation.webknossos.xyz/datasets/sample_organization/l4_sample/sandbox/skeleton
However, I think this condition was too broad and instead should only check that that no sandbox annotation is active. You should be able to do something like this `state.temporaryConfiguration.controlMode !== ControlModeEnum.SANDBOX`

Reply:
> I was curious why this condition was introduced (since the version view seems to work just fine for read-only annotations) and checked using git blame. Turns out, I introduced this when sandbox annotations were implemented. For sandbox annotations, the version view doesn't work, it cannot be closed again, for example 🙈 You can check using a sandbox URL, like https://allowversionhistoryinnotownedannotation.webknossos.xyz/datasets/sample_organization/l4_sample/sandbox/skeleton


Thanks for paying attention to this 🚀